src/scripts/templates/variant/Input Vector default template.js

```diff
@@ -41,19 +45,17 @@ function parseParameters(helper, msg) {
 
 function setParameter(helper, msg, param, value, escaped) {
     size = helper.getParamNumber();
-    if (size > 1) {         
-        query = helper.getParamValue(size);
-        
-        for (var i = 0; i < size; i++) {
-            pname = getParamName(i);
-            pvalue = getParamValue(i);                
-            if (pname == param) {      
-                    pvalue = helper.encodeBase64(value);
-            }
-                    
-            query = pname + "=" + pvalue + "&" + query;
+    query = helper.getParamValue(size-1);
+
+    for (var i = 0; i < size; i++) {
+        pname = helper.getParamName(i);
+        pvalue = helper.getParamValue(i);
+        if (pname == param) {
+                pvalue = helper.encodeBase64(value);
```

Comment:
The return value should be URL encoded (it's calling `setEscapedQuery`).

Reply:
Can change to `pvalue = helper.encodeBase64(encodeURI(value));`